### ISSUE TITLE
check, etc. shouldn't be circled

### ISSUE THREAD

**Issue by corysimmons** [thesabbir/simple-line-icons#98]
`check`, `plus`, `close`, etc. should just be the icon w/o circle.
`check-circle`, etc. should be the circled version.

Would need a major version bump for this change.

**Comment by MadHamza** [comment]
I agree and am glad if somebody can provide a css hack to remove the circle for a particular icon.
